website/src/middlewares/apiThrottle.js

```diff
@@ -0,0 +1,22 @@
+var nconf = require('nconf');
+var limiter = require('connect-ratelimit');
+
+var IS_PROD = nconf.get('NODE_ENV') === 'production';
+
+module.exports = function(app) {
+  if (!IS_PROD) return;
+  app.use(limiter({
+    end:false,
+    catagories:{
```

Comment:
Noticed the typo here, though it exists in the original as well. Does connect-ratelimit require the incorrect spelling? Otherwise, we should probably get this fixed.